frontend/src/pages/SaveView/__test__/SaveView.test.tsx

```diff
@@ -0,0 +1,172 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+/* eslint-disable sonarjs/no-duplicate-string */
+import ROUTES from 'constants/routes';
+import { explorerView } from 'mocks-server/__mockdata__/explorer_views';
+import { server } from 'mocks-server/server';
+import { rest } from 'msw';
+import { MemoryRouter, Route } from 'react-router-dom';
+import { fireEvent, render, screen, waitFor, within } from 'tests/test-utils';
+
+import SaveView from '..';
+
+const handleExplorerTabChangeTest = jest.fn();
+jest.mock('hooks/useHandleExplorerTabChange', () => ({
+	useHandleExplorerTabChange: () => ({
+		handleExplorerTabChange: handleExplorerTabChangeTest,
+	}),
+}));
+
+jest.mock('react-router-dom', () => ({
+	...jest.requireActual('react-router-dom'),
+	useLocation: jest.fn().mockReturnValue({
+		pathname: `${ROUTES.TRACES_SAVE_VIEWS}`,
+	}),
+}));
+
+describe('SaveView', () => {
+	it('should render the SaveView component', async () => {
+		render(<SaveView />);
+		expect(await screen.findByText('Table View')).toBeInTheDocument();
+
+		const savedViews = screen.getAllByRole('row');
+		expect(savedViews).toHaveLength(2);
+
+		// assert row 1
+		expect(
+			within(document.querySelector('.view-tag') as HTMLElement).getByText('T'),
+		).toBeInTheDocument();
+		expect(screen.getByText('test-user-1')).toBeInTheDocument();
+
+		// assert row 2
+		expect(screen.getByText('R-test panel')).toBeInTheDocument();
+		expect(screen.getByText('test-user-test')).toBeInTheDocument();
+	});
+
+	it('explorer icon should take the user to the related explorer page', async () => {
+		render(
+			<MemoryRouter initialEntries={['/traces/saved-views']}>
+				<Route path="/traces/saved-views">
```

Comment:
nit: can we replace `/traces/saved-views` with `ROUTES.TRACES_SAVE_VIEWS`? :slightly_smiling_face:

Reply:
Done